WMF Framework/Remote Notifications/RemoteNotificationsOperationsController.swift

```diff
@@ -52,6 +56,9 @@ class RemoteNotificationsOperationsController: NSObject {
             stop()
             return
         }
+        guard UserDefaults.wmf.didMakeAuthorizedTitleDescriptionEdit else {
+            return
```

Comment:
The `completion` should be called anywhere there's an early return (here and above)